bf2fc6cc711aee1a0c2a/kas-fleet-manager
# Mgdstrm 5111

## Description
https://issues.redhat.com/browse/MGDSTRM-5111 - Create public OpenAPI spec for the template

## Verification Steps
1. clone the repo
2. run `make openapi/generate` - make sure that no new files are generated
3. run `make run` (db setup and secrets setup will be required due to renaming of them) - make sure that the service starts. It will produce some erros in the workers, but this will be fixed in the subsequent Jiras (db schema, workers, tests, etc)
4. Run `find . -type f -name '*kafka*'` and make sure that there are no files with *kafka* in their name (apart from migrations which will be handled in separate Jira)
 
## Checklist (Definition of Done)
